test/usage/dune-sigterm/Makefile

```diff
@@ -0,0 +1,23 @@
+.PHONY : test-daemon
+test-daemon :
+	make clean
+	./test-daemon.sh
+	test -f bisect*.coverage
+
+.PHONY : test-normal
+test-normal :
+	make clean
+	dune exec ./normal.exe
+	test -f bisect*.coverage
+
+
+## TODO: test environment variable as well
+
+
+.PHONY : test
+test : test-normal test-daemon
+
+.PHONY : clean
+clean :
+	dune clean
+	rm -f bisect*.coverage
```

Comment:
Is the choice to create a usage test (intended so far for testing build systems) deliberate, as opposed to creating a cram test?

Reply:
nope, that was an oversight from my part. in the b425545 i move it to a cram test. i also changed the sleep thing (it was fickle) and replaced it with the fork suggestion of valentin. i was not sure in which folder to put the test, i've put it in `test/sigterm/` but i can move it somewhere else that makes more sense.